static/styles/deckviewer_styles.css

```diff
@@ -32,13 +32,103 @@ body.frogtown {
   background-image: url("https://www.frogtown.me/Images/CardBack.jpg");
 }
 
-#root{
+#root {
   display: flex;
   flex-direction: row;
   height: calc(100% - 64px);
 }
 
-#deckArea{
+#altPane {
+  visibility: hidden;
+  position: fixed;
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  height: 100%;
+}
+
+#altContainer {
+  height: calc(100% - 130px);
+  margin: 35px;
+  margin-top: 100px;
+  position: fixed;
+  border-radius: 5px;
+  background-color: #dfdfdf;
+}
+
+#altRenderArea {
+  height: calc(100% - 44px);
+  width: calc(100% - 8px);
+  position: relative;
+  top: 40px;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  overflow: auto;
+  padding-bottom: 48px;
+}
+
+#altDim {
+  position: fixed;
+  height: 100%;
+  width: 100%;
+  background-color: #000000;
+  opacity: 35%;
+}
+
+#altPaneCardArea{
+  height: calc(100% - 60px);
```

Comment:
This doesn't hide or scroll it's overflow-y, so there's no need to specify the height as anything. Same for the width, there's no need to specify it, it will default to 100% because this is a div. Also it's weird to shift the right border using both a subtraction on the width, and a right padding. Just combine those into one.